solutions/scripts/avn-shutoff-services.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/zsh
```

Comment:
since most of your script isn't specific to zsh, I would recommend change it to bash so this is friendly to users with default bash shell, such as fedora and etc.
`#!/bin/bash`